Ballerina.lock

```diff
@@ -1,19 +1,4 @@
 org_name = "wso2"
 version = "0.10.0"
```

Comment:
Can you change the org_name to ballerinax and update the version as well?

Reply:
Let's keep org_name as wso2 for this. since we need to do another release from the previous version with a deprecated message. yes, we need to update to a version.

Reply:
Done